### PR TITLE
Update the 'Guide to the documentation' section with more learning resources

### PR DESCRIPTION
**Pull request by rmartinezduque**
I've suggested replacing the "Guide to the documentation" title with the "Learning Resources" one, and added more content to this section. Instead of providing users with just an overview of the docs, I think it makes sense to share more information about the available learning materials (and add a link to the `Learn` page), so that users that land directly in the tutorial can find them easily. Please feel free to review it and ensure that you’re happy with these changes.

### CODE REVIEW

File: tutorial/part9-next-steps/guide-to-the-documentation.md [hunk 1]
```diff
@@ -1,9 +1,40 @@
...
+
+## Blog posts
+
+Other development posts, as well as case studies and general news, can be found in the [**Frontity blog**](https://frontity.org/blog/). Here's some that you might find interesting:
+
+- [Connecting Gutenberg and Frontity](https://frontity.org/blog/connecting-gutenberg-and-frontity/)
+- [How to use Frontity with SpinupWP](https://frontity.org/blog/how-to-use-frontity-with-spinupwp/)
+- [Diariomotor Case Study](https://frontity.org/blog/diariomotor-case-study/)
+
+## Showcase
```
Comment (juanmaguitar) [inline tutorial/part9-next-steps/guide-to-the-documentation.md:38]: If we add this "showcases" block to the "Learning Resources" subsection I guess we should also remove the latest "Showcases" subsection (under "Next Steps" section)

**Review by juanmaguitar — CHANGES_REQUESTED** [review]

File: tutorial/part9-next-steps/guide-to-the-documentation.md (hunk 1 above)
Comment (rmartinezduque) [reply tutorial/part9-next-steps/guide-to-the-documentation.md:38]: Yes, that's something that I wanted to suggest as well. I don't think the 'Showcases' subsection makes sense anymore. Could you remove that page yourself? Or do you prefer me to open an issue?

**Review by juanmaguitar — APPROVED** [review]

File: tutorial/part9-next-steps/guide-to-the-documentation.md (hunk 1 above)
Comment (juanmaguitar) [reply tutorial/part9-next-steps/guide-to-the-documentation.md:38]: I have added that change (removing this extra showcases page) to this PR (branch)

**Comment by juanmaguitar** [comment]
Approved & Merged!
Thsnks @rmartinezduque

**Comment by rmartinezduque** [comment]
Thank you @juanmaguitar! Looks like the title of the 'Guide to the documentation' didn't change accordingly. And the Showcase subsection is still there but empty. Could you take a look whenever you can? Thnx!

**Comment by juanmaguitar** [comment]
> Looks like the title of the 'Guide to the documentation' didn't change accordingly. And the Showcase subsection is still there but empty. 

@rmartinezduque Fixed! With Gitbook the `SUMMARY.md` content is the one deciding the title of the pages and the "render" of them (even if theres no existing `.md`)

**Comment by rmartinezduque** [comment]
Awesome, many thanks @juanmaguitar!

**Pull request merged**